### PR TITLE
fix anchors

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

anchor `#browser_action_2` hasn't been existed in page `Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension` ever since the file was imported into git (https://github.com/mdn/content/blob/cbe151a06d6e5b4d1fbb46081bd16e69ef4c1630/files/en-us/mozilla/add-ons/webextensions/anatomy_of_a_webextension/index.html)

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/extension/getbackgroundpage/index.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ function foo() {
 }
 ```
 
-A script running in a [popup](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#browser_actions_2) can call this function directly like this:
+A script running in a [popup](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2) can call this function directly like this:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/extension/getbackgroundpage/index.md:43]:
```suggestion
A script running in a [popup](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Popups) can call this function directly like this:
```

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/getbackgroundpage/index.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ function foo() {
 }
 ```
 
-A script running in a [popup](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#browser_actions_2) can call this function directly like this:
+A script running in a [popup](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2) can call this function directly like this:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/getbackgroundpage/index.md:59]:
```suggestion
A script running in a [popup](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Popups) can call this function directly like this:
```

File: files/en-us/mozilla/add-ons/webextensions/api/tabs/saveaspdf/index.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that
 
 ## Examples
 
-In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#browser_actions_2), then tries to save the currently active tab as a PDF file, then logs the result:
+In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2), then tries to save the currently active tab as a PDF file, then logs the result:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/tabs/saveaspdf/index.md:39]:
```suggestion
In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Toolbar_button), then tries to save the currently active tab as a PDF file, then logs the result:
```

File: files/en-us/mozilla/add-ons/webextensions/api/tabs/printpreview/index.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that
 
 ## Examples
 
-In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#browser_actions_2), then opens print preview for the currently active tab:
+In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2), then opens print preview for the currently active tab:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/tabs/printpreview/index.md:34]:
```suggestion
In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Toolbar_button), then opens print preview for the currently active tab:
```

File: files/en-us/mozilla/add-ons/webextensions/api/tabs/print/index.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ None.
 
 ## Examples
 
-In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#browser_actions_2), then tries to print the currently active tab:
+In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2), then tries to print the currently active tab:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/tabs/print/index.md:28]:
```suggestion
In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Toolbar_button), then tries to print the currently active tab:
```

File: files/en-us/mozilla/add-ons/webextensions/api/tabs/connect/index.md [hunk 6]
```diff
@@ -41,7 +41,7 @@ browser.tabs.connect(
 
 ## Examples
 
-In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#browser_actions_2), then connects to the currently active tab, then sends a message using the `Port` that's returned from `connect()`:
+In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2), then connects to the currently active tab, then sends a message using the `Port` that's returned from `connect()`:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/tabs/connect/index.md:44]:
```suggestion
In this example a background script listens for a click on a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Toolbar_button), then connects to the currently active tab, then sends a message using the `Port` that's returned from `connect()`:
```

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/messagesender/index.md [hunk 7]
```diff
@@ -33,7 +33,7 @@ Values of this type are objects. They contain the following properties:
...
+    If the sender is a script running in an extension page (such as a [background page](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#background_scripts), an [options page](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#options_pages), or a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2) or [page action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#page_actions) popup), the URL is in the form `"moz-extension://<extension-internal-id>/path/to/page.html"`. If the sender is a background script and you haven't included a background page, it is `"moz-extension://<extension-internal-id>/_generated_background_page.html"`.
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/messagesender/index.md:36]:
```suggestion
    If the sender is a script running in an extension page (such as a [background page](/en-US/docs/Mozilla/Add-ons/WebExtensions/Background_scripts), an [options page](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Options_pages), or a [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Toolbar_button) or [page action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Page_actions) popup), the URL is in the form `"moz-extension://<extension-internal-id>/path/to/page.html"`. If the sender is a background script and you haven't included a background page, it is `"moz-extension://<extension-internal-id>/_generated_background_page.html"`.
```

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/getbackgroundpage/index.md [hunk 8]
```diff
@@ -9,7 +9,7 @@ browser-compat: webextensions.api.runtime.getBackgroundPage
...
+This provides a convenient way for other privileged extension scripts to get direct access to the background script's scope. This enables them to access variables or call functions defined in that scope. "Privileged script" here includes scripts running in [options pages](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#options_pages), or scripts running in [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#sidebars_popups_and_options_pages_2) or [page action](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#page_actions) popups, but does _not_ include [content scripts](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#content_scripts).
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/getbackgroundpage/index.md:12]:
```suggestion
This provides a convenient way for other privileged extension scripts to get direct access to the background script's scope. This enables them to access variables or call functions defined in that scope. "Privileged script" here includes scripts running in [options pages](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Options_pages), or scripts running in [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Toolbar_button) or [page action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Page_actions) popups, but does _not_ include [content scripts](/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts).
```

**Review by rebloor — COMMENTED** [review]
@PassionPenguin I also noticed that the `page_actions` anchor is missing. Overall, given that these are links from detailed content, it probably makes more sense to link to the detailed content rather than the extension anatomy overview.

**Comment by PassionPenguin** [comment]
> @PassionPenguin I also noticed that the `page_actions` anchor is missing. Overall, given that these are links from detailed content, it probably makes more sense to link to the detailed content rather than the extension anatomy overview.

that's cool! thanks for your investigation!

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
Thanks @PassionPenguin

**Pull request merged**